nautobot/docs/development/apps/migration/migrating-an-app-from-v1.md

```diff
@@ -0,0 +1,191 @@
+# Migration Guide to Upgrade an App from V1 to V2
+
+## Model Updates
+
+### Core
+
+#### Replace the Usage of Slugs with Composite Keys
+
+Slugs were used to identify unique objects in the database for various models in Nautobot v1.x and they are now replaced by Composite Keys. The `slug` field can be safely deleted as long as your models are derived from `BaseModel` that automatically supports the following [natural key](https://docs.djangoproject.com/en/3.2/topics/serialization#natural-keys) APIs. For a more comprehensive guideline on how Natural Keys/Composite Keys in Nautobot v2.0 works, please go to the [Natural Key documentation](../../core/natural-keys.md).
+
+### DCIM
+
+#### Replace Site and Region with Location Model
+
+`Site` and `Region` Models are replaced by `Site` and `Region` `LocationTypes` and `Locations`. Your models and data that are associated with `Site` or `Region` via ForeignKey or ManyToMany relationships are now required to be migrated to `Locations`. Please go [here](region-and-site-to-location.md) for a comprehensive migration guide on how to migrate your data from `Site` and `Region` to `Location` and `LocationType`.
```

Comment:
may be? if not, I don't understand the sentence
```suggestion
`Site` and `Region` Models are replaced by `LocationTypes` and `Locations`. Your models and data that are associated with `Site` or `Region` via ForeignKey or ManyToMany relationships are now required to be migrated to `Locations`. Please go [here](region-and-site-to-location.md) for a comprehensive migration guide on how to migrate your data from `Site` and `Region` to `Location` and `LocationType`.
```

Reply:
Perhaps:

```suggestion
`Site` and `Region` Models are replaced by `LocationTypes` called `"Site"` and `"Region"` and existing `Site` and `Region` records will be replaced by `Location` records using these types. Your models and data that are associated with `Site` or `Region` via ForeignKey or ManyToMany relationships are now required to be migrated to `Locations`. Please go [here](region-and-site-to-location.md) for a comprehensive migration guide on how to migrate your data from `Site` and `Region` to `Location` and `LocationType`.
```